tests/models/test_evaluation.py

```diff
@@ -710,10 +793,60 @@ def test_evaluator_interface(multiclass_logistic_regressor_model_uri, iris_datas
                     run_id=run.info.run_id,
                     evaluator_config=evaluator1_config,
                     custom_metrics=None,
+                    baseline_model=None,
                 )
 
 
-def test_evaluate_with_multi_evaluators(multiclass_logistic_regressor_model_uri, iris_dataset):
+@pytest.mark.parametrize(
+    ("baseline_model_uri", "expected_error"),
+    [
+        (
+            "pyfunc",
+            pytest.raises(
+                MlflowException,
+                match=(
+                    "The baseline model argument must be a string URI "
+                    + "referring to an MLflow model"
+                ),
+            ),
+        ),
+        (
+            "invalid_model_uri",
+            pytest.raises(OSError, match="No such file or directory: 'invalid_uri'"),
+        ),
+    ],
+    indirect=["baseline_model_uri"],
+)
+def test_model_validation_interface_invalid_baseline_model_should_throw(
+    multiclass_logistic_regressor_model_uri, iris_dataset, baseline_model_uri, expected_error
+):
+    with mock.patch.object(
+        _model_evaluation_registry, "_registry", {"test_evaluator1": FakeEvauator1}
+    ):
+        evaluator1_config = {"config": True}
+        classifier_model = mlflow.pyfunc.load_model(multiclass_logistic_regressor_model_uri)
+        with expected_error:
+            evaluate(
+                classifier_model,
```

Comment:
I think this line needs to be fixed. `evaluate` no longer accepts a pyfunc model.

Reply:
Fixed!